dvc/exceptions.py

```diff
@@ -338,3 +338,18 @@ def __init__(self, target, file):
 
 class MergeError(DvcException):
     pass
+
+
+class CacheLinkError(DvcException):
+    SUPPORT_LINK = "See {} for more information.".format(
+        format_link(
+            "https://dvc.org/doc/user-guide/troubleshooting#cache-types"
+        )
+    )
+
+    def __init__(self, path_infos):
+        msg = "No possible cache link types for '{}'. {}".format(
```

Comment:
Used to be "no possible cache types left to try" which is more of a phrase. Is that still what we mean by "No possible cache link types"? It looks like a typo in "not possible" but even then the phrase wouldn't make that much sense. Sorry, not sure what to suggest, as I don't understand the conditions that could lead to this error.

Reply:
For `dvc checkout` it will occur in the same situations as the old "left to try" error. The "left to try" qualifier no longer seemed necessary to me given that we have the troubleshooting page w/a more thorough explanation now

Reply:
I see. What were those situations again? 🙂

Reply:
Oh I see it's explained in https://github.com/iterative/dvc.org/pull/1892#pullrequestreview-521909163:

> if your configured cache types are unsupported for your filesystem... or where a platform normally supports your configured link type(s) but the actual link operation itself fails due to some (non-DVC related) problem

Reply:
So how about something like

> Unable to create any of the configured cache link types: '{cache.type list}'.
> Affected files: '{list of outputs}'.
> See {link} for more details.